index.html

```diff
@@ -3,44 +3,148 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>Felony Disenfranchisement</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Fjalla+One|Exo:400,600|Ropa+Sans' rel='stylesheet' type='text/css'>
   <style>
     .header {
-      text-align: center;
+      text-align: left;
     }
+    table {
+    border-collapse: collapse;
+    width: 100px;
+    }
+
+    th, td {
+    padding: 8px;
+    text-align: left;
+    border-bottom: 1px solid #ddd;
+    }
+
+    tr:hover td{font-weight: bold;
+    }
+
+    body {
+      background-color: #4C4C4C;
+          }
+
+    th {
+       font-family: 'Exo', sans-serif;
```

Comment:
Note that sense you're using this font everywhere, it could be defined once in the `body` declaration, and everything else on the page would inherit it.